tests/infra/remote.py

```diff
@@ -189,7 +189,14 @@ def _setup_files(self):
             LOG.info("[{}] copy {} from {}".format(self.hostname, tgt_path, path))
         session.close()
 
-    def get(self, file_name, dst_path, timeout=FILE_TIMEOUT, target_name=None):
+    def get(
+        self,
+        file_name,
+        dst_path,
+        timeout=FILE_TIMEOUT,
+        target_name=None,
+        pre_condition_func=None,
```

Comment:
```suggestion
        pre_condition_func=lambda src_dir, ???: True
```

may be more readable and avoid a branch further down.